src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx

```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
 
 To find the file name for a specific integration, see the [integration documentation](/docs/infrastructure/host-integrations).
 
-## Update using apt (Debian or Ubuntu) [#apt]
+## Pick your update method based on your environment [#install]
+
+Your services may be running on a single physical host, on a VM, or in a container orchestrated by Kubernetes or ECS. Whatever the setup, our on-host integrations can adapt to your environment and send data to New Relic; all you have to do is choose the appropriate update method.
+
+### You're running services in orchestrated environments [#orchestrated]
```

Comment:
We have no support for windows integrations in orchestrated environments so no further info is needed here.

For linux the links are OK as upgrading is as a new install so they are relevant here.